src/odf/dataset-impl/src/entities/dataset_storage_unit_local_fs.rs

```diff
@@ -11,31 +11,27 @@ use std::path::PathBuf;
 use std::sync::Arc;
 
 use async_trait::async_trait;
-use dill::*;
 use internal_error::{ErrorIntoInternal, ResultIntoInternal};
 use odf_dataset::*;
 use odf_metadata::*;
 use odf_storage::*;
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+#[dill::component]
+#[dill::interface(dyn DatasetStorageUnit)]
+#[dill::interface(dyn DatasetStorageUnitWriter)]
```

Comment:
Note: in Q2/Q3 roadmap, this will stop being a component, S3 as well, as we will support multiple storage units.